bundle-size/test/app.test.js

```diff
@@ -182,38 +182,7 @@ describe('bundle-size', () => {
     ]);
   });
 
-  test('mark a check as successful when a capable user approves the PR',
-      async () => {
```

Comment:
Investigating this code removal from `eslint --fix`

Reply:
Update: There is no code removal, this is just GitHub/diffing getting confused because the block is changing its indentation.